linalgo/annotate/models.py

```diff
@@ -407,6 +409,33 @@ def __init__(
         self.annotator = Annotator(annotator)
         self.task = Task(task)
         self.reviewee = Annotator(reviewee)
-    
+
     def __repr__(self) -> str:
-        return f'Schedule::{self.type}::{self.status}'
\ No newline at end of file
+        return f'Schedule::{self.type}::{self.status}'
+
+
+@dataclass
+class Body:
```

Comment:
Defining body here will break existing implementations. It's probably best to have an abstract class defining the interface.